robottelo.properties.sample

```diff
@@ -271,3 +271,23 @@ ssh_key=
 # for RHEL7 - the ports needs to have a correct selinux context set:
 # 'semanage port -a -t websm_port_t -p tcp <port-range>'
 port_range=9091, 14999
+
+# section for real capsule
+# [capsule]
+# to setup a real capsule we need the capsule virtual machine hostname
+# resolvable.
+# we should register a hostname and get a hash value as described in
+# https://mojo.redhat.com/docs/DOC-13143, that we should use on the stage of
+# ddns setup
+# domain is the base ddns domain, the virtual machine hostname will be a sub
+# domain of this domain.
+# domain=
+# the immage_name is the key used in the registration above, and used to name
+# the image of the virtual machine
+# the real finale hostname should be image_name.domain
+# image_name=
```

Comment:
@kbidarkar, the image name here has no effect on what the real RHEL version image is being used but must provide information from wich sat version and wich sat rhel version is being tested , and can be rhel6 or rhel7.
Of course we can retrieve the server being used and automatically choose the image to be used RHEL6 or RHEL7, but this is not the requirement of the config (if you want we can discuss about this point).
Because all the tests of different platform can run in the same time and we do not want them to use the same hostname.
* so we have 3 sat version 6.1 6.2 and 6.3 and 2 platforms  rhel6 and rhel7
so we need 6 hostnames::
1. qe_61_rhel6_capsule
2. qe_61_rhel7_capsule  
3. qe_62_rhel6_capsule  
4. qe_62_rhel7_capsule  
5. qe_63_rhel6_capsule  
6. qe_63_rhel7_capsule